profile/README.md

```diff
@@ -0,0 +1,55 @@
+## Hi there, we are Fobnail Team 👋
+
+The Fobnail project provides the resources to create an axiomatically
+trustworthy device and a simple user interface to confirm the status of the
+platform. We aim to provide a reference architecture for building
+offline integrity measurement verifiers on a USB device and validators running
+in dynamically run measured environments (DLME).
+
+Our goal is to create a scalable, flexible implementation that will be the main
+building block of future (community) security solutions based on DRTM payloads.
+The owner of Fobnail can verify the reliability of the working system before
+performing any sensitive operation. Fobnail does not need an internet
+connection, which makes it immune to network stack attacks and remote
+infrastructure.
```

Comment:
Platform provisioning requires internet connection to obtain TPM certificates' chains as of now. Fobnail Token presents itself as a network device, so it uses network stack.

I don't get last part of this sentence. Fobnail is immune to remote infrastructure?

Reply:
Sentence was removed, also took information from fobnail documentation...